cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -298,7 +298,7 @@
      */
     public static final String API_VERSION_3_TOKEN = "v3";
     public static final String API_VERSION_3 = "/" + API_VERSION_3_TOKEN;
-    public static final String STREAM_HANDLER_NAME = "stream.rest";
+    public static final String STREAM_HANDLER_NAME = "stream_rest";
```

Comment:
due to how queries happen (using period in the queryParams), the metrics Query handler splits on period, so this messes it up.